### ISSUE TITLE
[Charm v1.3.2/Minecraft 1.12.2] Config does not disable options disabled

### ISSUE THREAD

**Issue by RoofToilet1107** [svenhjol/Charm#149]
I have noticed that ALL of the options I disabled are still available in-game, appear in JEI, and are craftable. Below is a pastebin link to a copy/paste of my charm.cfg file and a list of the features that still appear in-game. I checked each of the features to make sure they actually work as if they were enabled, and they do. Any help? Is this user error, or is this an issue with Charm. I really don't want to have to remove Charm because I absolutely adore it.

Forge: 14.23.5.2838
Quark: r1.6-176
JEI: 4.15.0.268

Redstone Sand
Variable Redstone Lamp
Decay
Bookshelf Chest
Rotten Flesh Blocks
Suspicious Soup
Composter
Homing Enchantment (w/ hoe)
Extracting XP from books
Tallow Increasing Durability
Gold Tool Improvements
Cauldron Infinite Water Source
Sheared Melons
Sponge Reducing Fall Damage
Water In Hot Biomes
Witch Hat Protection
Charged Emeralds
Spectres and Spectre Haunting

Pastebin: https://pastebin.com/V2b9qjWT

**Comment by RoofToilet1107** [comment]
Additionally, I should mention, the "config" button is grayed out for Charm in the mods menu,

**Comment by svenhjol** [comment]
I started a world with Charm 1.3.2, all config features enabled so it registered all blocks and items. Then I restarted with your attached configuration and everything that you set to `false` deregistered as expected. Here's a screenshot showing the output:

![Screenshot from 2019-08-13 14-24-20](https://user-images.githubusercontent.com/26497903/62949714-39b00a00-bdd6-11e9-93e9-0547bd42f7d7.png)

As an example I searched for "redston" and does not show any registered redstone sand from Charm in JEI.

![Screenshot from 2019-08-13 14-27-49](https://user-images.githubusercontent.com/26497903/62949926-9f03fb00-bdd6-11e9-88fd-8aca514356b6.png)

I'm not sure what is causing your error, but maybe the config file is in the wrong place or named incorrectly. Try deleting the config file completely, let it regenerate a new one, then replace the contents with the file you attached and see if that changes things.

The greyed out config button is intended as Charm does not offer any GUI-based configuration at this time.
